src/lib/InterventionList.svelte

```diff
@@ -51,10 +52,10 @@
     currentHover.set(id);
   }
 
-  function onKeydown(e) {
+  function onKeydown(e: KeyboardEvent) {
     const id = $currentlyEditing;
     if (e.key == "Delete") {
-      const tag = e.originalTarget.tagName;
+      const tag = (e.target as HTMLElement).tagName;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/originalTarget
Oops, we were using a Firefox-only property before! TS to the rescue.